lib/resources/example_json/r4_examples_appointment.rb

```diff
@@ -372,6 +372,18 @@ module Resources
         'op': 'replace',
         'path': '/status',
         'value': 'cancelled'
+      },
```

Comment:
since this is a new operation, this should be created separately from the status patch.